### PR TITLE
Feature/jordan/travis-and-readme

### PR DESCRIPTION
**Pull request by jordan-moore**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,21 +2,33 @@
...
-* python-dotenv
-* slackclient
+* websocket-client==0.47.0
+* python-dotenv>=0.9.1
+* requests>=2.19.1
+* slackclient>=1.2.1
 
 ---
 
+### Getting Your Slack Key
```
Comment (troyibm) [inline README.md:20]: I set up a meeting for tomorrow to go through this part as I don't see these choices in my slack workgroup.

**Comment by troyibm** [comment]
@jordan-moore there was a `.env.enc` file added.  I assume that was a mistake?

**Comment by troyibm** [comment]
@jordan-moore @JordanTMoore you have two public github logins??

**Comment by jordan-moore** [comment]
@troyibm hmm I could remove it, it's used in the travis testing automation, could be useful if we ever decided to add travis to the public repo

**Comment by troyibm** [comment]
No, that's fine.  Encoded env is used for travis.  Got it.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+language: python
+python:
+- '3.6'
```
Comment (troyibm) [inline .travis.yml:3]: I know that indentation matters in a yml file.  Some of these lines starting with hyphen are indented and some aren't (line 3 isn't, but 5 is).  Does this file really work?

File: README.md (hunk 1 above)
Comment (troyibm) [reply README.md:20]: had meeting and determined steps to be 

1. Go to https://YOUR_SLACK.slack.com/apps/manage
2. In the __Search App Directory__ field at the top, type __bots__
3. Click on __Bots__
4. Click on the __Add Configuration__ button
5. Give your bot a unique __username__ (Save this info)
6. Save the __API Token__
7. Create your __.env__ file as instructed in the below steps

File: .travis.yml (hunk 2 above)
Comment (jordan-moore) [reply .travis.yml:3]: yes

**Review by troyibm — APPROVED** [review]

**Pull request merged**